### ISSUE TITLE
关于mask prompt的训练方法question about mask prompt

### ISSUE THREAD

**Issue by sove45** [MedicineToken/Medical-SAM2#60]
我注意到您在论文中有提到maskprompt，可是代码中没有体现，请问怎么使用第一帧的mask作为prompt来微调神经网络呢
I noticed that you mentioned 'maskprompt' in the paper, but it was not reflected in the code. Could you explain how to use the mask from the first frame as a prompt to fine-tune the neural network
